modules/dr-restoring-etcd-quorum-ha.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * disaster_recovery/quorum-restoration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="dr-restoring-etcd-quorum-ha_{context}"]
+= Restoring etcd quorum for high availability clusters
+
+You can use the `quorum-restore.sh` script to instantly brings back a new single-member etcd cluster based on its local data directory and marks all other members as invalid by retiring the previous cluster identifier. No prior backup is required to restore the control plane from.
+
+[WARNING]
+====
+You might experience data loss if the host that runs the restoration does not have all data replicated to it.
+====
+
+.Prerequisites
+
+* You have SSH access to the node used to restore quorum.
+
+.Procedure
+
+. Select a control plane host to use as the recovery host. This is the host that you run the restore operation on.
+
+. Using SSH,  connect to the chosen recovery node and run the following command to restore etcd quorum:
++
+[source,terminal]
+----
+$ sudo -E /usr/local/bin/quorum-restore.sh
+----
+
+. Exit the SSH session.
+
+. Wait until the control plane recovers by running the following command:
++
+[source,terminal]
+----
+$ oc adm wait-for-stable-cluster
+----
+
+.Troubleshooting
+
+If you see no progress rolling out the etcd static pods, you can force redeployment from the `cluster-etcd-operator` by running the following command:
```

Comment:
```suggestion
If you see no progress rolling out the etcd static pods, you can force redeployment from the `cluster-etcd-operator` pod by running the following command:
```

I believe `cluster-etcd-operator` is pod. 

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references

> An object reference is when you are referring to the actual instance of an API object, where the object name is important.